x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/Authenticator.java

```diff
@@ -159,6 +164,13 @@ public SecureString getBearerString() {
             return bearerString;
         }
 
+        public SecureString getApiKeyString() {
```

Comment:
nit: can you include a mention of this in the commit/pr message

Reply:
Ok.
But TBC, this change was not a purpose in itself. I introduced it to be able to use a method call to check if any APIKey credential can be extracted from the `Authorization` header, in `AuthenticatorChain#shouldFallbackToAnonymous` below.
Previously that duplicated the header string parsing which obscured the "has an API Key been used" check?